deploy/libsimple_init

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+WORKDIR=/var/www/circulation
+BINDIR=${WORKDIR}/bin
+
+if [ -d /etc/circulation ] && ! [ -z "$LIBSIMPLE_DB_INIT" ] && [ -f ${BINDIR}/util/initialize_instance ]; then\
```

Comment:
The check for `/etc/circulation` ensures the script will only run when a container is being created with configuration. The check for the variable allows it to be passed in with the command `docker run -e LIB_SIMPLE_INIT=yes` and the last criteria is the check for the script, which isn't included in any existing circulation releases (but probably will be in the next).